.eslintrc.js

```diff
@@ -75,8 +75,8 @@ module.exports = {
         'import/prefer-default-export': 0,
         'import/extensions': ['warn', 'never', { jsx: 'always', json: 'always' }],
         'no-sequences': ['warn'],
-        'import/no-unresolved': [2, { ignore: ['@deriv/components', '@deriv/shared'] }],
-
+        'import/no-unresolved': [2, { ignore: ['@deriv/components', '@deriv/shared', '@deriv/components/cashier'] }],
+        // 'import/no-unresolved': 0,
```

Comment:
please remove this commented code

Reply:
> 

Hi @ali-hosseini-fs Thank you , Updated the code and fixed the conflicts.